File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.js
function solution(s) {
    let count = {};
    for(let i=0; i<s.length; i++){
        let char = s[i];
        if(count[char]){
            count[char]++;
        }else{
            count[char] = 1;
        }
    }
    
    let answer = [];
    for(let key in count){
        if(count[key] === 1){
            answer.push(key);
        }
    }
    return answer.sort().join('');
}

File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.py
def solution(s):
    answer = ''
    
    str = list(s)
    check = set(str)
    
    for i in check:
        cnt = 0
        for num in str:
            if num == i:
                cnt += 1
        if cnt <=1:
            answer += i
    
    return ''.join(sorted(answer))
        
